edgelight/edgelight.ino

```diff
@@ -38,218 +36,226 @@ SOFTWARE.
 #include "Base64.h"
 
 #ifdef ESP32
```

Comment:
can get rid of this too.  this code is designed for a pretty specific platform and requires SPIFFS.